src/WebJobs.Script.WebHost/Configuration/ServerlessSecurityDefenderOptionsSetup.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Az.ServerlessSecurity.Platform;
+using Microsoft.Azure.WebJobs.Script.Config;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Configuration
+{
+    internal class ServerlessSecurityDefenderOptionsSetup : IConfigureOptions<ServerlessSecurityDefenderOptions>
+    {
+        private readonly IEnvironment _environment;
+
+        public ServerlessSecurityDefenderOptionsSetup(IEnvironment environment)
+        {
+            _environment = environment;
+        }
+
+        public void Configure(ServerlessSecurityDefenderOptions options)
+        {
+            options.EnableDefender = _environment.GetEnvironmentVariable("SERVERLESS_SECURITY_ENABLED") != null;
```

Comment:
Where do we set this environment variable? Is it while creating the container? How about in windows (i.e. non-container environment)?

Reply:
I see. For our testing, where do we set it? In the AppSettings?

Reply:
Yes, in the AppSettings.